### ISSUE TITLE
Failed to install Simple Heels

### ISSUE THREAD

**Issue by Fleimi** [Caraxi/SimpleHeels#38]
Download for 0.9.3.2 fails when trying to install through Dalamud from the SeaOfStars repo. Caraxion repo does not work even to the point of being able to see the plugin available for install.

**Comment by DTV96Calibre** [comment]
TLDR: Update XIV Launcher and edit your .desktop file replacing `XIVLauncher.Core` with `xivlauncher-core`

If you are like me, this issue may have to do with OpenSSL being more strict about security and refusing the connection due to some deficiency in ssl on repo.caraxion.com, which appears to also be used for downloading the Simple Heels plugin in the SeaOfStars repo. This appears to be addressed by newer versions of xivlauncher, but your .desktop file might need to be edited, replacing `XIVLauncher.Core` with `xivlauncher-core`. You can find the path to your .desktop file by opening XIVLauncher, opening Settings > Debug and looking for something like `GIO_LAUNCHED_DESKTOP_FILE`. 

Caraxi might fix this by updating to a more secure ssl implementation, and probably should, seeing as a fix from anywhere else is just going to ignore the insecurities and allow the insecure connection.

I'm no SSL expert and have only gleaned this information like anyone else would.

**Comment by Ozc-Y** [comment]
Are you playing on Linux and/ or Steam Deck? If so, issue #36 has the solution: Add `http://repo.caraxian.com/no_https.json` as a custom plugin repo. It is indeed a hiccup with SSH.

**Comment by oz-orz** [comment]
Are you playing on Linux and/ or Steam Deck? If so, issue #36 has the solution: Add `http://repo.caraxian.com/no_https.json` as a custom plugin repo.

**Comment by Fleimi** [comment]
Thanks, I was able to find this solution on my own as well on the Mare Discord. Not really good with github so I was unable to spot that issue that had the fix.

Could I suggest to add a line or two about it to the readme to help the next person with this issue?

**Comment by DTV96Calibre** [comment]
Before this gets closed, adding that as a custom plugin repo didn't resolve the issue for me.

**Comment by dblackwell80** [comment]
I'm on Windows 11, I have added all 3 repositories, the one from sea of stars, the  caraxion one, and the caraxion with no https. I get the install error failed for all 3. I'm not quite sure how to proceed to get this to install. Is there anyone that can help? I opened the xiv launcher settings and I dont see any Debug section to edit the .desktop file either. Any help is greatly appreciated, thank you!


![install_error](https://github.com/Caraxi/SimpleHeels/assets/111915026/fbb40823-a456-4d6d-b969-b18558a31672)

**Comment by Caraxi** [comment]
> I'm on Windows 11, I have added all 3 repositories, the one from sea of stars, the caraxion one, and the caraxion with no https. I get the install error failed for all 3. I'm not quite sure how to proceed to get this to install. Is there anyone that can help? I opened the xiv launcher settings and I dont see any Debug section to edit the .desktop file either. Any help is greatly appreciated, thank you!
> 
> ![install_error](https://private-user-images.githubusercontent.com/111915026/341798473-fbb40823-a456-4d6d-b969-b18558a31672.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkwMDAxNTcsIm5iZiI6MTcxODk5OTg1NywicGF0aCI6Ii8xMTE5MTUwMjYvMzQxNzk4NDczLWZiYjQwODIzLWE0NTYtNGQ2ZC1iOTY5LWIxODU1OGEzMTY3Mi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyMVQxOTU3MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YjE4MWM2MWI1NTBhZjI1MTE3ZDBhZDFmYWM2NWY0YWFiMGVkNTA5YTE2ZTM0NjdlNDVhMzBlMDVmOWVmNmY1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.loSscfFydjfbM3gxmX6KN2dX52TZyq4GF3r1C1KYLu8)

Please share your dalamud log file
![image](https://github.com/Caraxi/SimpleHeels/assets/11886749/ac869ef5-02a9-4fd8-8809-23d55aae9858)

**Comment by dblackwell80** [comment]
Hello! Thank you for the reply. I have attached the dalamud.log file here. I attempted to run the repo again this morning from both Caraxian as well as SeaOfStars, so those are the most recent error entries in the log file. Thank you for any help!
[dalamud.log](https://github.com/user-attachments/files/15955925/dalamud.log)

**Comment by dblackwell80** [comment]
When I try to access that zip file, it's telling me "insecure file blocked" for some reason.

**Comment by Caraxi** [comment]
thats because its a http link from a https website

**Comment by dblackwell80** [comment]
is there a way for me to be able to fix this so I can download the plugin?
